bundles/org.openhab.binding.sensorcommunity/src/main/resources/OH-INF/i18n/sensorcommunity.properties

```diff
@@ -0,0 +1,47 @@
+# add-on
+
+addon.sensorcommunity.name = sensorcommunity Binding
```

Comment:
Currently this does not correspond to the actual name which is "SensorCommunity Binding", but this file probably will need to be regenerated anyway after changes.